src/de/grobox/blitzmail/send/SendActivity.java

```diff
@@ -184,8 +186,18 @@ private void sendAttachment(@Nullable JSONObject jMail) {
 		}
 
 		try {
-			String num = String.valueOf(jMail.getJSONArray("attachments").length());
-			jMail.put("subject", num + " " + getString(R.string.files_shared) + " " + getString(R.string.app_name));
+			JSONArray files = jMail.getJSONArray(MAIL_ATTACHMENTS);
+			String filename = files.getJSONObject(0).getString("filename");
```

Comment:
Thanks for using the `MAIL_ATTACHMENTS` constant here. Do we also have one for `filename`?

Reply:
Good point. There was one for MAIL_SUBJECT which I just updated but there isn't one for filename.

I'd be happy to define a scheme or constants for attachments but are you sure this should be done in this pull request?

Reply:
Can be in another one, just brought it up since you started to use constants.